### DISCUSSION TITLE
How to change the size of the profile photo?

### DISCUSSION THREAD

**Discussion by chang-cai** [alshedivat/al-folio#1066] [Q&A] [answered]
Hello, I got stuck in changing the size of the profile photo. Your help is highly appreciated!

Moreover, the icons are a little bit big for me. How can I make it smaller?

**Comment by alshedivat** [comment, marked as answer]
you will need to edit CSS styles to make these changes. specifically:
- change `width: 30%` here (https://github.com/alshedivat/al-folio/blob/master/_sass/_base.scss#L117) to any other size you want.
- change `font-size: 4rem` here (https://github.com/alshedivat/al-folio/blob/master/_sass/_base.scss#L256) to control the size of the social icons at the bottom of the page.

**Reply by Asonjay** [reply]
Hi, for changing the image size, the first solution is not working anymore. I tried to add ``width:30%`` inside the image tag, the image scale is correct now, but the profile image is not justified to the right. Can you look into this?

**Reply by george-gca** [reply]
Can you share the changes you made?

**Reply by Asonjay** [reply]
Thank you for your reply! First, I found @alshedivat 's reply does not make sense because the line number he sent me is not related to the image tag. Thus I made the changes by locating these tags in "_base.scss":

```
figure,
img {
  max-width: 55vw;
  width: 40%; # this is added line
}
```

Before adding the width, the page looks like this:

![image](https://github.com/alshedivat/al-folio/assets/49351464/ce718517-6333-4229-81d2-722dd6c68388)

After adding width:40%, it looks like this:

![image](https://github.com/alshedivat/al-folio/assets/49351464/dd1919a0-6a67-4f4e-bf60-25cb47f7e199)

**Reply by george-gca** [reply]
Note that his answer was from years ago, so the line his answer was pointing to changed. Doing a quick search for `width: 30%` in the file [_sass/_base.scss](https://github.com/alshedivat/al-folio/blob/master/_sass/_base.scss) pointed me [here](https://github.com/alshedivat/al-folio/blob/e10c45cdfec9b262fcf3918636b869e9a27eb5e2/_sass/_base.scss#L220).

**Reply by Asonjay** [reply]
Thank you for your reply! It worked :)

**Reply by sjtu-iisc** [reply]
Is it possible to make the profile picture in the main page and the one in the people page with different width? thanks!

**Reply by george-gca** [reply]
For this you'll have to change the [profile page layout](https://github.com/alshedivat/al-folio/blob/f4a6e184a9a23c2a2070b0ea545d390fab1e5c98/_layouts/profiles.liquid#L18).
